paper-input-container.html

```diff
@@ -515,8 +515,7 @@
     _handleValue: function(inputElement) {
       var value = this._inputElementValue;
 
-      // type="number" hack needed because this.value is empty until it's valid
```

Comment:
And here's a JSBin that uses _your_ branch, for comparison: http://jsbin.com/suruje/edit?html,output

Type `111` and then an `e` in the `paper-input` to repro.